@glenn-jocher I have tested on all JetPack versions. JetPack5 and 6 are all good. However, when I try on JetPack4, the following error is there:

```sh
Step 14/17 : RUN python3 -m pip install --upgrade pip uv
 ---> Running in c071ed461d7f
Collecting pip
  Downloading https://files.pythonhosted.org/packages/ef/7d/500c9ad20238fcfcb4cb9243eede163594d7020ce87bd9610c9e02771876/pip-24.3.1-py3-none-any.whl (1.8MB)
Collecting uv
  Downloading https://files.pythonhosted.org/packages/ca/1c/8c40ec75c26656bec9ada97833a437b49fd443b5d6dfd61d6dda8ad90cbe/uv-0.5.7.tar.gz (2.4MB)
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
    ModuleNotFoundError: No module named 'setuptools'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-z7znwiky/uv/
The command '/bin/sh -c python3 -m pip install --upgrade pip uv' returned a non-zero code: 1
```

I have pushed a commit for a workaround where we install `uv` separately. Then the problem is gone and the image gets built. This could be related to maybe JetPack4 image using Python3.8 vs others using Python3.10+.

--- a/ultralytics/__init__.py
+++ b/ultralytics/__init__.py
@@ -1,6 +1,6 @@
 # Ultralytics YOLO 🚀, AGPL-3.0 license
 
-__version__ = "8.3.48"
+__version__ = "8.3.49"
 
 import os
 

